File: Python/tests/test_gng_m1.py
import pytest

from hbayesdm.models import gng_m1


def test_gng_m1():
    _ = gng_m1(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_gng_m1.R
context("Test gng_m1")
library(hBayesDM)

test_that("Test gng_m1", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(gng_m1(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
